freud/box.pyx

```diff
@@ -300,13 +304,18 @@ cdef class Box:
         vecs = np.asarray(vecs)
         flatten = vecs.ndim == 1
         vecs = np.atleast_2d(vecs)
-        vecs = freud.util._convert_array(vecs, shape=(None, 3)).copy()
+        vecs = freud.util._convert_array(vecs, shape=(None, 3))
+        out = freud.util._convert_array(out, shape=vecs.shape)
 
         cdef const float[:, ::1] l_points = vecs
```

Comment:
Let's rename `l_points` to `l_vecs` to match the variable name used elsewhere.

Reply:
Just to clarify: do you mean the ```l_points``` in ```box.wrap``` or all the  ```l_points``` in ```box.pyx```?

Reply:
Nevermind, I see there are lots of exceptions. I would have liked the C++ and Python parameter names to match, e.g. `const vec3<float>* vecs` in C++ and `l_vecs` / `vecs` in Cython. It doesn't really matter, this comment can be ignored. We could make an issue to refactor all of that to improve readability but the benefit is small.